api/app.py

```diff
@@ -20,37 +20,37 @@
 from core.util import LanguageCodes
 from flask.ext.babel import Babel
 
-app = Flask(__name__)
-
-testing = 'TESTING' in os.environ
-db_url = Configuration.database_url(testing)
-# Initialize a new database session unless we were told not to
-# (e.g. because a script already initialized it).
-autoinitialize = os.environ.get('AUTOINITIALIZE') != 'False'
-if autoinitialize:
-    SessionManager.initialize(db_url)
-session_factory = SessionManager.sessionmaker(db_url)
-_db = flask_scoped_session(session_factory, app)
-if autoinitialize:
-    SessionManager.initialize_data(_db)
 
+app = Flask(__name__)
+app._db = None
```

Comment:
app._db needs to be initialized before it can handle requests, but it doesn't have to happen right now.